packages/cozy-client/src/models/timeseries.js

```diff
@@ -64,16 +65,17 @@ export const saveTimeSeries = async (client, timeseriesOption) => {
  * Helper to retrieve time series by their date interval and source.
  *
  * @param {object} client - The CozyClient instance
- * @param {{ startDate, endDate, dataType, source, limit }} The query params.
+ * @param {TimeSeries} timeSeries - The query params.
  *
  * @typedef TimeSeriesJSONAPI
  * @property data {Array<TimeSeries>} - The JSON-API data response
  * @returns {Promise<TimeSeriesJSONAPI>} The TimeSeries found by the query in JSON-API format
  */
 export const fetchTimeSeriesByIntervalAndSource = async (
   client,
-  { startDate, endDate, dataType, source, limit }
+  timeSeries
```

Comment:
To describe destructured object, you can : 
```
/**
@param {string} arg1
@param {string} options.arg2
@param {string} options.arg3

const func(arg1, { arg2, arg3 }) 

**/

```

It should make the trick. See https://jsdoc.app/tags-param.html#parameters-with-properties

Reply:
Did the trick, change has been done.